types/student/student.enum.ts

```diff
@@ -0,0 +1,16 @@
+export enum TypeWork {
+  'Praca w biurze' = 1,
+  'Praca zdalna' = 2,
+}
+
+export enum ContractType {
+  'Umowa o pracę' = 1,
+  'B2B' = 2,
+  'Umowa zlecenie' = 3,
+  'Umowa o dzieło' = 4,
+}
+
+export enum Internship {
+  'Nie',
+  'Tak',
+}
```

Comment:
Uważam, że te enumy można poprawić. Napisałem jak na FE.
W dodatku, czemu te same typy są 2x, zamiast zaciągać je z BE na FE?